source/reference/class/MongoDBModelCollectionInfo.txt

```diff
@@ -8,8 +8,8 @@ Definition
 .. phpclass:: MongoDB\Model\CollectionInfo
 
    This class models information about a collection. Instances of this class are
-   returned by traversing a :phpclass:`MongoDB\Model\CollectionInfoIterator`,
-   which is returned by :phpmethod:`MongoDB\Database::listCollections()`.
+   returned in an `Iterator <{+php-manual}/class.iterator.php>`__
```

Comment:
S: [nit] since this sentence uses "returned" twice, you could change to "stored"
```suggestion
   stored in an `Iterator <{+php-manual}/class.iterator.php>`__
```

Reply:
Applies to all